### PR TITLE
Adding properties file

### PR DESCRIPTION
**Pull request by Red-GV**
### Description
This PR adds the properties file to the bundle. The properties file is the location where dependencies and specific properties (like the maximum supported OpenShift version) can be defined.

/assign @jcantrill

### CODE REVIEW

File: bundle/metadata/properties.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+properties:
```
[the remark this answers is not in the document]
Comment (Red-GV) [reply bundle/metadata/properties.yaml:1]: As far as I know, the `properties.yaml` file goes here. The `operator-sdk` doesn't seem to be able copy or generate this file while it is making the bundle (it seems to only make the `annotations.yaml`).

[the remark this answers is not in the document]
Comment (jcantrill) [reply bundle/metadata/properties.yaml:1]: I believe you could add it here https://github.com/openshift/cluster-logging-operator/tree/master/config/manifests and ref'd in the appropriate kustomize but if it wont be removed then maybe not an issue

**Pull request merged**